docs/source/getting_started/datasets.rst

```diff
@@ -0,0 +1,109 @@
+  It also includes requests of the platform: some API calls will be slower if there are more datasets.
+
+Datasets can be listed and gotten from projects just as projects are from the client itself:
+
+.. code-block:: python
+
+    project = citrine.projects.get("baaa467e-1758-43a8-97c7-76e569d0dcab")
+    datasets_in_project = project.datasets.list()
+    a_dataset = project.datasets.get("6ed8bf3b-876b-40f0-9d50-982e686e5cd3")
+
+Datasets can only be deleted if they are empty.
+If you need to delete a non-empty dataset, see this discussion on :ref:`deleting data objects <deleting_data_objects_label>`.
+
+Dataset Access and Sharing
+--------------------------
+
+When a dataset is created on the Citrine platform, only members of the project in which it was created can see it and interact with it.
+If a dataset is made public it can be retrieved by any user using any project.
+Once retrieved, its data objects are also available to view.
+Datasets that are public may only be written to by the project from which they originated.
+
+Toggling Public Access
+^^^^^^^^^^^^^^^^^^^^^^
+
+A dataset can be made public (or made private again) using the project from which it originates
+using the :func:`~citrine.resources.project.Project.make_public` and :func:`~citrine.resources.project.Project.make_private` methods on the project.
+
+Example
+^^^^^^^
+
+Assume you have a "band gaps project" with known ID, `band_gaps_project_id`, and an associated dataset with known ID, `strehlow_cook_dataset_id`.
+
+Making a dataset public:
+
+.. code-block:: python
+
+    band_gaps_project = citrine.projects.get(band_gaps_project_id)
+    strehlow_cook_dataset = band_gaps_project.datasets.get(strehlow_cook_dataset_id)
+
+    # Make the Strehlow and Cook dataset publicly accessible so that it can be retrieved
+    # from any project
+    band_gaps_project.make_public(strehlow_cook_dataset)
+
+Making a dataset private:
+
+.. code-block:: python
+
+    band_gaps_project = citrine.projects.get(band_gaps_project_id)
+    strehlow_cook_dataset = band_gaps_project.datasets.get(strehlow_cook_dataset_id)
+
+    # If the Strehlow and Cook dataset was previously publicly available, revoke that
+    # access so that it can only be retrieved and from the band_gaps_project.
+    band_gaps_project.make_private(strehlow_cook_dataset)
+
+Files
+-----
+
+In addition to data objects, a dataset can also contain files.
+These could be images output by a microscope in a proprietary format, a picture of how several samples are arranged on a hot plate, or anything else you would like to save.
+The association of a file with a resource is done using a :class:`FileLink <citrine.resources.file_link.FileLink>`, which is created when you upload a file.
+The `FileLink` can be associated with runs, specs, attributes and templates, all of which have a `file_links` field, and it can be used to download the file.
```

Comment:
```suggestion
The ``FileLink`` can be associated with runs, specs, attributes and templates, all of which have a ``file_links`` field, and it can be used to download the file.
```